website/content/en/platforms/arm.adoc

```diff
@@ -96,7 +105,7 @@ Note that not all peripherals are supported on all CPUs.
 * SBC6045 with Atmel at91sam9g45
 * Technologic Systems TS-7200 and TS-7800
 * Wandboard
-* Zedboard (Xilinx Zynq)
+* Zedboard (Xilinx Zynq).
 
 Not all peripherals are supported on all boards.
```

Comment:
This paragraph seems superfluous, repetitive. 

[`d5ecf98` (#215)](https://github.com/freebsd/freebsd-doc/pull/215/commits/d5ecf982d667f8895b4831acd55cffceeb222a6c#diff-06532acdcf84a18734f78e5ee58d1ee12f58c3353c542a6b57eac4e3fe71615bR56) forewarns that _not every peripheral is supported on every CPU or board_.